db/migrate/20220916091250_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
```

Comment:
i guess there is no datatype string in mysql

Reply:
ok